### PR TITLE
foreach

### PR DESCRIPTION
**Pull request by AntKarpo**

### CODE REVIEW

**Comment by AntKarpo** [comment]
i have done all challenges. back to the grind

File: JavaScript/ArrayMethod/js-array-methods_for-each/js/index.js [hunk 1]
```diff
@@ -19,4 +19,19 @@ const colors = [
   "#A5A5A5",
 ];
 
-colors.forEach();
+colors.forEach((color)=> {
+  const hex = document.createElement("div");
```
Comment (reverbePrintemps) [inline JavaScript/ArrayMethod/js-array-methods_for-each/js/index.js:23]: It's a little strange to me that you're naming your `div` "hex" 🤔

File: JavaScript/ArrayMethod/js-array-methods_for-each/js/index.js [hunk 2]
```diff
@@ -19,4 +19,19 @@ const colors = [
...
+  hex.style.background = color;
+  document.body.append(hex);
+});
+
+
+function renderColorBox(color){
+  const hex = document.createElement("div");
+  hex.classList.add('color-box');
+  hex.style.background = color;
+  document.body.append(hex);
```
Comment (reverbePrintemps) [inline JavaScript/ArrayMethod/js-array-methods_for-each/js/index.js:34]: A template string could be a more elegant solution. Especially if you would have to add more nodes, classes, attributes, etc.

File: JavaScript/ArrayMethod/js-array-methods_repetition/index.js [hunk 3]
```diff
@@ -35,31 +35,38 @@ const recipes = [
...
 
 // HINT: use first map() and then Array.prototype.join() with "method chaining" like so:
 // myArray.map(...).join(', ');
 
-const allTitlesInOneString = null; // 'Crepes, Scrambled Eggs, ...'
-
+const allTitlesInOneString = recipes.map((res)=>{ // 'Crepes, Scrambled Eggs, ...'
+return res.title;
+});
+console.log(allTitlesInOneString.join());
```
Comment (reverbePrintemps) [inline JavaScript/ArrayMethod/js-array-methods_repetition/index.js:58]: This is wrong. If you run `npm run test` you'll see your test failing.

**Review by reverbePrintemps — CHANGES_REQUESTED** [review]
Mostly ok except for failing test, you might want to look into that before merging.

**Pull request merged**